web-client/public/js/update-app.js

```diff
@@ -127,25 +176,251 @@ const synchronizeHideAllWeights = () => {
+              </a>
+            </li>`;
+    };
+
+    for (var property in network.expression) {
+        if (property.match(ENDS_IN_EXPRESSION_REGEXP)) {
+            grnState.nodeColoring.nodeColoringOptions.push({value: property});
+        }
+    }
+
+    $(BOTTOM_DATASET_SELECTION_SIDEBAR).append($("<option>")
+            .attr("value", "Same as Top Dataset").text("Same as Top Dataset"));
+
+    $(BOTTOM_DATASET_SELECTION_MENU).append(createHTMLforDataset("Same as Top Dataset"));
+
+    grnState.nodeColoring.nodeColoringOptions.forEach(function (option) {
+        var shortenedSheetName = shortenExpressionSheetName(option.value);
+        $(TOP_DATASET_SELECTION_SIDEBAR).append($("<option>")
+              .addClass("dataset-option")
+              .attr("value", option.value).text(shortenedSheetName));
+        $(TOP_DATASET_SELECTION_MENU)
+              .append(createHTMLforDataset(option.value));
+        $(BOTTOM_DATASET_SELECTION_SIDEBAR).append($("<option>")
+              .addClass("dataset-option")
+              .attr("value", option.value).text(shortenedSheetName));
+        $(BOTTOM_DATASET_SELECTION_MENU)
+              .append(createHTMLforDataset(option.value));
+    });
+
+    $("#topDatasetDropdownMenu li a span").first().addClass("glyphicon-ok");
+    $("#bottomDatasetDropdownMenu li a span").first().addClass("glyphicon-ok");
+};
+
+const updateLogFoldChangeMaxValue = () => {
+    var value = logFoldChangeMaxValueInputValidation(grnState.nodeColoring.logFoldChangeMaxValue);
+    $(LOG_FOLD_CHANGE_MAX_VALUE_SIDEBAR_INPUT).val(value);
+    $(LOG_FOLD_CHANGE_MAX_VALUE_MENU).val(value);
+    updaters.renderNodeColoring();
+};
+
+const removeAllChecksFromMenuDatasetOptions = (id) => {
+    $(`${id} li a span`).each(
+        function () {
+            $(this).removeClass("glyphicon-ok");
+        }
+    );
+};
+
+const updateTopDataset = () => {
+    $(TOP_DATASET_SELECTION_SIDEBAR).val(grnState.nodeColoring.topDataset);
+    removeAllChecksFromMenuDatasetOptions(TOP_DATASET_SELECTION_MENU);
+    $(`${TOP_DATASET_SELECTION_MENU} li[value='${grnState.nodeColoring.topDataset}'] a span`).addClass("glyphicon-ok");
+    updaters.renderNodeColoring();
+    // TO DO: If bottomDataSameAsTop make bottom selction "Same As Top"
+};
+
+const updateBottomDataset = () => {
+    if (grnState.nodeColoring.bottomDataSameAsTop) {
+        $(BOTTOM_DATASET_SELECTION_SIDEBAR).val("Same as Top Dataset");
+        removeAllChecksFromMenuDatasetOptions(BOTTOM_DATASET_SELECTION_MENU);
+        $(`${BOTTOM_DATASET_SELECTION_MENU} li[value='${"Same as Top Dataset"}'] a span`).addClass("glyphicon-ok");
+    } else {
+        $(BOTTOM_DATASET_SELECTION_SIDEBAR).val(grnState.nodeColoring.bottomDataset);
+        removeAllChecksFromMenuDatasetOptions(BOTTOM_DATASET_SELECTION_MENU);
+        /* eslint-disable max-len */
+        $(`${BOTTOM_DATASET_SELECTION_MENU} li[value='${grnState.nodeColoring.bottomDataset}'] a span`).addClass("glyphicon-ok");
```

Comment:
I won’t let this block an approval, but isn’t this avoidable? Just move the `.addClass` to the next line, and indented to show that it is a function call on the `$` call’s result.
```suggestion
        $(`${BOTTOM_DATASET_SELECTION_MENU} li[value='${grnState.nodeColoring.bottomDataset}'] a span`)
            .addClass("glyphicon-ok");
```

`eslint` directives should really be last last last last resorts, when there is a completely unavoidable reason to avoid the linting error.